composer.json

```diff
@@ -11,10 +11,12 @@
             "email": "dcramer@gmail.com"
         }
     ],
+    "minimum-stability": "dev",
     "require-dev": {
         "friendsofphp/php-cs-fixer": "~2.1",
         "phpunit/phpunit": "^4.8 || ^5.0",
-        "monolog/monolog": "*"
+        "nokitakaze/testhttpserver": "^0.1.0",
```

Comment:
>the curl stuff has 100% been broken before, so unless we've got a great way to mock it then i think this is fine

As I said below, all the code related to cURL is going to be removed in the near future as we agreed to delegate the part of code that handles how events are sent to external libraries like HttpPlug so that users are free to choose whatever transport method they prefer. This means that the fact that cURL (in this case) will already be tested by the 3rd party library and we will only need unit tests, not integration tests